docs/install/index.md

```diff
@@ -3,36 +3,29 @@ html_meta:
   "description": "Install Plone 6"
   "property=og:description": "Install Plone 6"
   "property=og:title": "Install Plone 6"
-  "keywords": "Plone 6, install, installation, installing, buildout, pip, docker"
+  "keywords": "Plone 6, install, overview"
 ---
 
 (install-index-label)=
 
 # Install Plone 6
 
-This section focuses on installing Plone 6 for the developer, tester, or contributor.
 
-For trying out Plone 6, visit our official demonstration site located at:
+(install-index-getting-started-label)=
 
-```{toctree}
-:maxdepth: 2
-:hidden: true
+## Getting started
 
-containers/index
-```
+* What do you want to do?
+  * Try a demo now
+  * Run Plone in containers (why use containers here..)
+  * Develop Plone add-ons or contribute to Plone packages → Install Plone from source
 
-https://6.demo.plone.org/
 
-```{todo}
-For deployment of Plone 6, visit TBD.
-```
-
-
-(install-index-system-requirements-pre-requisites-label)=
+(install-index-system-requirements-label)=
 
-## System requirements and pre-requisites
+## System Requirements
 
-To install Plone 6, you must satisfy system requirements and pre-requisites.
+To install Plone 6, you must satisfy system requirements.
```

Comment:
To install Plone 6 from source, you must satisfy system requirements.

Reply:
See above comment. This page is for general system requirements that are not specific to an installation method, assuming that even exists.

Reply:
See 51f3f20a64781919bd3e7ba5c90530c1497bd6fc